README.md

```diff
@@ -42,3 +42,7 @@ Number             | Title                                                    |
 [32](bsip-0032.md) | Always Match Orders At Maker Price                                          | Abit More | Protocol | Draft
 [33](bsip-0033.md) | Maker Orders With Better Prices Take Precedence                             | Abit More | Protocol | Draft
 [34](bsip-0034.md) | Always Trigger Margin Call When Call Price Above Or At Price Feed           | Abit More | Protocol | Draft
+[36](bsip-0036.md) | Remove expired feeds                                                        | oxarbitrage | Protocol | Draft
```

Comment:
To be clearer, how about "Remove expired price feeds on maintenance interval"?

Reply:
changed, sent pull.